pkg/apis/networking/types.go

```diff
@@ -48,30 +48,44 @@ const (
 
 // NetworkPolicySpec provides the specification of a NetworkPolicy
 type NetworkPolicySpec struct {
-	// Selects the pods to which this NetworkPolicy object applies. The array of
-	// ingress rules is applied to any pods selected by this field. Multiple network
-	// policies can select the same set of pods. In this case, the ingress rules for
-	// each are combined additively. This field is NOT optional and follows standard
-	// label selector semantics. An empty podSelector matches all pods in this
-	// namespace.
+	// Selects the pods to which this NetworkPolicy object applies.
+	// A network flow is bidirectional but initiated in one direction,
+	// from a source to a destination.
+	// The whole flow is allowed or not depending on whether the initiation is allowed.
+	// A network flow with a pod on either or both ends is allowed to be initiated
```

Comment:
so you're leaving `ipBlock` policies undefined...

Reply:
A NetworkPolicy necessarily applies to a pod, right?  It is in the characterization of the peer that IP blocks are used.  The existing language covers the case of a connection between a pod and a non-pod.